### ISSUE TITLE
How to perform single image detection？

### ISSUE THREAD

**Issue by EatonL** [Jingtao-Li-CVer/TDD#1]
May I ask if it is possible to directly load training weights for single image detection？

**Comment by Jingtao-Li-CVer** [comment]
Of course. If you want to infer directly, you can just change the configuration of the test image in the config file (test section under the dataset keyword). Which config file to use depends on which training image you want.

**Comment by Jingtao-Li-CVer** [comment]
> May I ask if it is possible to directly load training weights for single image detection？

Of course. If you want to infer directly, you can just change the configuration of the test image in the config file (test section under the dataset keyword). Which config file to use depends on which training image you want.
